Packs/Base/Scripts/ValidateContent/ValidateContent.yml

```diff
@@ -56,6 +56,6 @@ timeout: 600ns
 runonce: false
 runas: DBotWeakRole
 fromversion: 5.5.0
-dockerimage: demisto/xsoar-tools-hf:1.0.0.31061
+dockerimage: devdemisto/xsoar-tools:1.0.0.55361
```

Comment:
devdemisto?

Reply:
will change this when the docker is merged